lib/multicell_location/services/nrf_cloud_integration.c

```diff
@@ -149,12 +149,13 @@ int location_service_get_cell_location_nrf_cloud(
 		.rx_buf_len = rcv_buf_len,
 		.fragment_size = 0
 	};
-	const struct nrf_cloud_rest_cell_pos_request loc_req = {
-		.net_info = (struct lte_lc_cells_info *)params->cell_data
+	const struct nrf_cloud_rest_location_request loc_req = {
+		.cell_info = (struct lte_lc_cells_info *)params->cell_data,
+		.wifi_info = NULL
```

Comment:
No a topic of this PR, but it would be convenient if a lib with an API like multicell_location is having, could also serve wifi in a future.